app/controllers/users_controller.rb

```diff
@@ -1,4 +1,12 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :admin_user, only: :destroy
+
+  def index
+    @users = Kaminari.paginate_array(User.all).page(params[:page])
+      .per(Settings.show_limit.show_10)  
```

Comment:
Use `` @users = User.page(params[:page]).per Settings.show_limit.show_10``

Reply:
It exception: undefined method `page' for #Class:0x007f57b5c253f8
I have edited in several ways, but not edited

Reply:
I was found how to fix. :))